articles/the-future-of-device-management-ep1.md

```diff
@@ -25,5 +25,5 @@ In today’s episode of the Future of Device Management podcast, we speak with [
 <meta name="authorGitHubUsername" value="zwass">
 <meta name="authorFullName" value="Zach Wasserman">
 <meta name="publishedOn" value="2022-06-06">
-<meta name="articleTitle" value="Future of device management">
+<meta name="articleTitle" value="Future of device management episode 1">
```

Comment:
@mike-j-thomas is this an issue?

Reply:
I just saw it while I was in there to make them look consistent from fleetdm.com/podcasts
<img width="1349" alt="image" src="https://user-images.githubusercontent.com/78363703/180572682-56395a8a-0278-448e-b35e-962dd265b77b.png">